tiledb/sm/array_schema/array_schema.cc

```diff
@@ -779,10 +780,21 @@ ArraySchema ArraySchema::deserialize(ConstBuffer* buff, const URI& uri) {
     }
   }
 
+  // Populate timestamp range
+  std::pair<uint64_t, uint64_t> timestamp_range;
+  st = utils::parse::get_timestamp_range(uri, &timestamp_range);
```

Comment:
Nit for follow-up: we should be returning the `std::pair` here rather than `Status` (and throw inside the function if needed).